src/client/components/ProfileCareerResult/ProfileCareerResult.styles.css

```diff
@@ -0,0 +1,49 @@
+.robo-text {
```

Comment:
what is this? Robot font?

Reply:
Yeah, actually I found it on @sofiiadidovych CSS file:
https://github.com/HackYourFuture-CPH/Guide-IT/blob/f8f1eed1c08f2b434d3fe3f7474f387c278a34a2/src/client/components/ProfileQuizResults/ProfileQuizResults.styles.css

Reply:
@dpfernandes and @Poorooshka I added this class to my header, which is next to robot logo (that`s why I named it robo-text) to place it at the center of the image: 
![image](https://user-images.githubusercontent.com/71496144/118483246-11291280-b716-11eb-96c9-8bdf942d88f0.png)
I can rename it, if it is confusing.

Reply:
Thanks. I used the same so our components look alike.

Reply:
let's add a T there then :)

Reply:
✅ Done
https://github.com/HackYourFuture-CPH/Guide-IT/pull/158